app/views/users/show.html.erb

```diff
@@ -20,7 +21,7 @@
 <div class="user_show">
 	<% @user.answers.each do |answer| %>
 	<ul>
-	  <li><%= answer.content %></li>
+	  <li><%= link_to "#{answer.content}", question_path(answer.question_id) %></li>
```

Comment:
Same can be done here too, just:

<%= link_to "#{answer.content}", answer.question %>